src/views/ManageList.jsx

```diff
@@ -49,13 +38,13 @@ export function ManageList() {
 					/>
 				</div>
 				<select
-					onChange={(e) => setItemDuration(e.target.value)}
+					onChange={(e) => setItemDuration(Number(e.target.value))}
```

Comment:
I am wrapping `Number` here because `e.target.value` returns a string and we need it as a number.

Reply:
So number converts `Number("4")` into `4` as an integer